### DISCUSSION TITLE
Deploy YOLOv8 on NVIDIA Jetson using TensorRT and DeepStream SDK | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#198] [Q&A]
# Deploy YOLOv8 on NVIDIA Jetson using TensorRT and DeepStream SDK | Seeed Studio Wiki

Deploy YOLOv8 on NVIDIA Jetson using TensorRT and DeepStream SDK

https://wiki.seeedstudio.com/YOLOv8-DeepStream-TRT-Jetson/

**Comment by sergii-matiukha** [comment]
How to make INT8 engine for yolov8 model with imgsz=3040,  for use in deepstream python app on jetson? And how much does the accuracy drop when switching from Fp32 to INT8?

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by JCastel358** [comment]
Hello, i have a question and im pretty new into machine vision and using jetson devices in general. would it be feasible to run YOLOv8 on a jetson nano for a real time project like a drone being able to catch a thrown ball in mid-air?

**Reply by MatthewJeffson** [reply]
Hi, based on the wiki, it is feasible and you can install Jectpack 4.6 or higher on your Jetson Nano first, to implement it.

**Comment by antoineTK** [comment]
Hi all!

In chapter "INT8 Calibration", isn't there a step missing consisting of creating the calibration table file?

**Reply by mmmstew** [reply]
I think so too. I follow the steps carefully and get "File does not exist: /home/[user]/DeepStream-Yolo/calib.table". Did you find a solution?

**Reply by antoineTK** [reply]
no I did not
